.babelrc

```diff
@@ -1,3 +1,23 @@
 {
```

Comment:
"@babel/plugin-proposal-class-properties" was added to transpile ES6 arrow functions used within React components mainly